### PR TITLE
Remove cache implementation

### PR DESCRIPTION
**Pull request by jirevwe**

### CODE REVIEW

File: database/postgres/event.go [hunk 1]
```diff
@@ -160,18 +157,21 @@ const (
...
+		LEFT JOIN convoy.events_endpoints ee ON ev.id = ee.event_id
+		WHERE ev.deleted_at IS NULL
 	`
 
 	baseCountPrevEventSearch = `
-	SELECT COUNT(DISTINCT(ev.id)) AS COUNT
-	FROM convoy.events_search ev
-	LEFT JOIN convoy.events_endpoints ee ON ev.id = ee.event_id
-	WHERE ev.deleted_at IS NULL
+	select exists(
```
Comment (mekilis) [inline database/postgres/event.go:168]: They are equivalent, yeah?

File: services/create_endpoint.go [hunk 2]
```diff
@@ -12,15 +12,14 @@ import (
...
-	"github.com/frain-dev/convoy/cache"
 	"github.com/frain-dev/convoy/datastore"
 	"github.com/frain-dev/convoy/pkg/log"
 	"github.com/frain-dev/convoy/util"
 	"github.com/oklog/ulid/v2"
 )
 
 type CreateEndpointService struct {
-	Cache          cache.Cache
+	// Cache          cache.Cache
```
Comment (mekilis) [inline services/create_endpoint.go:22]: Should be deleted

**Review by mekilis — COMMENTED** [review]
It's ok. Left two comments.

File: database/postgres/event.go (hunk 1 above)
Comment (jirevwe) [reply database/postgres/event.go:168]: Yep

**Review by mekilis — APPROVED** [review]
Good to go.

**Pull request merged**